standalone/include/psen_scan_v2_standalone/protocol_layer/scanner_state_machine_def.h

```diff
@@ -131,23 +131,76 @@ inline void ScannerProtocolDef::sendStopRequest(const T& event)
   args_->control_client_->write(data_conversion_layer::stop_request::serialize());
 }
 
-inline void ScannerProtocolDef::printUserMsgFor(const ScanValidatorResult& res)
+inline void ScannerProtocolDef::printUserMsgFor(const ScanRound::Result& res)
 {
-  using Result = ScanValidator::Result;
-  if (!res || res.value() == Result::valid)
+  if (res == ScanRound::Result::started_new_round_early)
   {
-    return;
+    if (args_->config_.fragmentedScansEnabled())
+    {
+      PSENSCAN_WARN("StateMachine",
+                    "Detected dropped MonitoringFrame."
+                    " (Please check the ethernet connection or contact PILZ support if the error persists.)");
+    }
+    else
+    {
+      PSENSCAN_WARN("StateMachine",
+                    "Detected a MonitoringFrame from a new scan round before the old one was complete."
+                    " The uncomplete scan round will be dropped."
```

Comment:
```suggestion
                    " The incomplete scan round will be dropped."
```
Just as a remark since I suspect you are still changing these lines.